module/utils/species-command-parser.js

```diff
@@ -79,21 +79,27 @@ export async function CreateMonParser(input, andCreate = false) {
     return commands;
 }
 
-export async function GetSpeciesArt(mon, imgDirectoryPath, type = ".png") {
+export async function GetSpeciesArt(mon, imgDirectoryPath, type = ".png", shiny = false) {
     const basePath = imgDirectoryPath+(imgDirectoryPath.endsWith('/') ? '' : '/')
 
-    let path = basePath+lpad(mon?.number, 4)+type;
+    let shiny_path = "";
+    if(shiny)
+    {
+        shiny_path = "s";
+    }
```

Comment:
```suggestion
    const shiny_path = shiny ? "s" : "";
```
Not using null coalescing operator here because realitstically the 'shiny' path could be `""` or the default `false`.

Should maybe instead have this shiny path be set from a variable, like 
```js
const shiny_path = shiny === true ? "s" : shiny ? shiny : "";
```